doc/source/reference/routines.indexing.rst

```diff
@@ -41,6 +41,7 @@ Indexing-like operations
    diag
    diagonal
    select
+   as_strided
```

Comment:
I think this needs to be `lib.stride_tricks.as_strided`.

But anyways, yes, you can definitely test this with a local docs build (`cd doc` followed by `make html`).